src/main/java/ToDo.java

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Tasks that are used for ToDo.
+ */
+
+public class ToDo extends Task {
+
+    public ToDo(String description, boolean isDone) {
+        super(description, isDone);
+    }
+
+    @Override
+    public String getType() {
+        return "T";
+    }
+
+    @Override
+    public String getDate() {
```

Comment:
I'm not pretty sure why a dummy getDate() method is needed in here? It might be a better idea to remove getDate from Task class and implements it only in relevant Task such as Deadline and Event.

Reply:
Ohh, I made my abstract Task class implement an abstract getDate() method, hence, I need a dummy getDate() method for todo. Thanks for pointing it would will remove it!